test/system/src/test/java/io/pravega/test/system/AbstractScaleTests.java

```diff
@@ -37,19 +37,16 @@
     @Getter(lazy = true)
     private final ConnectionFactory connectionFactory = new ConnectionFactoryImpl(ClientConfig.builder().build());
     @Getter(lazy = true)
-    private final ControllerImpl controller = createController();
+    private final ClientFactoryImpl clientFactory = new ClientFactoryImpl(SCOPE, new ControllerImpl(
```

Comment:
Could we use the `controller` attribute as input for the `clientFactory`?

Reply:
removed redundant changes.